packaging/README.md

```diff
@@ -0,0 +1,45 @@
+# Packaging and Distributing Ignite
+
+Ignite CLI is distributed on multiple platforms and package managers. This document describes how to package and distribute Ignite CLI.
+
+## Flatpak
+
+Read the folowing resources to understand Flatpak.
+
+* <https://docs.flathub.org/docs/for-app-authors>
```

Comment:
I'm getting 404 from https://docs.flathub.org/docs/for-app-authors is it not valid or deprecated?